src/include/miopen/solver.hpp

```diff
@@ -108,6 +108,8 @@ struct SolverBase
     /// run-time parameters.
     virtual bool IsDynamic() const { return false; }
 
+    static constexpr float wti_unknown = -2;
```

Comment:
[notice] google C++ style recommends camelCase with prefix "k", i.e. `kWtiUnknown` (which is very like a kind of hungarian notation which I personally hate). I would prefer WTI_UNKNOWN for this specific case. Decide yourself, no feedback required.

Reply:
Caps is generally reserved for macros to indicate caution required when interacting with them. But I change it to any style if you want.

Reply:
Up to you, my comment is just a notice